__fixtures__/test-project/web/src/Redwood.stories.mdx

```diff
@@ -0,0 +1,667 @@
+import { Meta } from '@storybook/addon-docs'
```

Comment:
Do we need such a massive mdx file here? Just nitpicking, but a smaller fixture makes it easier to understand the text

Reply:
Cool, would you mind if we had a minimal md file here? Let me know if you’re short on time I can do it for you